docs/companion_guide.md

```diff
@@ -411,6 +411,31 @@ billion transactions per second (tps).
 | Schema / Data Representation | JSON Formatted |
 | Query Capabilities | [Not found in documentation] |
 
+| Peergos	   |  https://https://peergos.org/ |
+| -------------- | ---------------------------------------------------------- |
+| Description	   | Peergos is a decentralised protocol and open-source platform for storage, social media and applications |
+| Specification  | Proprietary specification |
+| Deployment     | Offered as a SaaS solution. Can be self hosted. |
+| Identity       | [PKI](https://book.peergos.org/security/pki.html) + [Centralized password management, with password based keypairs(password + username + public salt)](https://book.peergos.org/security/login.html) |
+| Authentication |  Self-authenticated & [S3 V4 Signatures](https://book.peergos.org/security/bats.html)|
+| Authorization  | [Cryptree](https://book.peergos.org/security/cryptree.html) based encryption and [Block access controls](https://book.peergos.org/security/bats.html) |
+| Transport      | [HTTP RESTful APIs](https://book.peergos.org/features/apps.html) |
+| Schema / Data Representation | [DAG CBOR Encoded IPLD Objects and Raw Objects](https://book.peergos.org/security/bats.html). JSON Schema for configuration. |
+| Query Capabilities | Peergos offers a RESTFul API with various capabilities described [here](https://book.peergos.org/features/apps.html). A few endpoints are directly specified. |
+
+
+| Decentralized Web Nodes	   | https://identity.foundation/decentralized-web-node/spec/ |
+| -------------- | ---------------------------------------------------------- |
+| Description	   | Decentralized Web Nodes are a mesh-like datastore construction that enable an entity to operate multiple nodes that sync to the same state across one another, enabling the owning entity to secure, manage, and transact their data with others without reliance on location or provider-specific infrastructure, interfaces, or routing mechanisms. |
+| Specification  |  [Open-Source Specification incubated by DIF](https://identity.foundation/decentralized-web-node/spec/) |
+| Deployment     | [ Pending ] |
```

Comment:
I think we can probably add here that it will be offered as a multi-tenant SaaS service, as well as self-hosted.